src/Argon/Argon.php

```diff
@@ -1,11 +1,12 @@
 <?php namespace Winter\Storm\Argon;
 
 use Carbon\Carbon as DateBase;
```

Comment:
On line 42 wouldn't it be more accurate to state that the return type is Argon, not DataBase? We could then get rid of the DateBase import.